### DISCUSSION TITLE
USB host support for NXP (specifically RT10xx)

### DISCUSSION THREAD

**Discussion by CrushedB** [zephyrproject-rtos/zephyr#78289] [Q&A] [answered]
Hello, 
Read the thread at issue [42066](https://github.com/zephyrproject-rtos/zephyr/issues/42066). 
Does NXP have an estimation when USB host support will be implemented. 
(Specifically I'm interested in the MSC class I should be able to plug in an USB thumb drive into board.  This is already available as an MCUXpresso SDK example app (I"m using [evkmimxrt1060_host_msd_fatfs_bm](https://github.com/nxp-mcuxpresso/mcux-sdk-examples/tree/main/evkmimxrt1060/usb_examples/usb_host_msd_fatfs))).

**Comment by ofirshe** [comment]
I'm following this discussion as I'm also interested in support for the USB host controller on the RT1060.

**Comment by CrushedB** [comment]
Initial support is in pull request [77973](https://github.com/zephyrproject-rtos/zephyr/pull/77973)

**Comment by DerekSnell** [comment, marked as answer]
Hi @CrushedB ,
You already found the RFC https://github.com/zephyrproject-rtos/zephyr/issues/42066 for adding USB Host support and the planned classes.  Zephyr Project has been working to add USB Host support, and that RFC is the best resource for the plans and progress.  

Best regards
